src/main/groovy/hpo/api/model/LoincTerm.groovy

```diff
@@ -0,0 +1,39 @@
+package hpo.api.model
+
+/**
+ * A minimum representation of a LOINC term
+ */
+class LoincTerm {
```

Comment:
This is a groovy class. They automatically have setters/getters attached based on the member variables no need to create them.